ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/InitializeMigrationCommand.java

```diff
@@ -0,0 +1,113 @@
+        + "  version_key  STRING KEY,\n"
+        + "  version      STRING,\n"
+        + "  name         STRING,\n"
+        + "  state        STRING,  \n"
+        + "  checksum     STRING,\n"
+        + "  started_on   STRING,\n"
+        + "  completed_on STRING,\n"
+        + "  previous     STRING\n"
+        + ") WITH (  \n"
+        + "  KAFKA_TOPIC='" + topic + "',\n"
+        + "  VALUE_FORMAT='JSON',\n"
+        + "  PARTITIONS=1,\n"
+        + "  REPLICAS= " + replicas + " \n"
+        + ");\n";
+  }
+
+  private String createVersionTable(final String name, final String topic) {
+    return "CREATE TABLE " + name + "\n"
+        + "  WITH (\n"
+        + "    KAFKA_TOPIC='" + topic + "'\n"
+        + "  )\n"
+        + "  AS SELECT \n"
+        + "    version_key, \n"
+        + "    latest_by_offset(version) as version, \n"
+        + "    latest_by_offset(name) AS name, \n"
+        + "    latest_by_offset(state) AS state,     \n"
+        + "    latest_by_offset(checksum) AS checksum, \n"
+        + "    latest_by_offset(started_on) AS started_on, \n"
+        + "    latest_by_offset(completed_on) AS completed_on, \n"
+        + "    latest_by_offset(previous) AS previous\n"
+        + "  FROM migration_events \n"
+        + "  GROUP BY version_key;\n";
+  }
+
+  @Override
+  public void run() {
+    final MigrationConfig properties = new MigrationConfig(
+        PropertiesUtil.loadProperties(new File("ksql-migrations.properties")));
+    final String eventStreamCommand = createEventStream(
+        properties.getKsqlMigrationsStreamName(),
+        properties.getKsqlMigrationsStreamTopicName(),
+        properties.getKsqlMigrationsTopicReplicas()
+    );
+    final String versionTableCommand = createVersionTable(
+        properties.getKsqlMigrationsTableName(),
+        properties.getKsqlMigrationsTableTopicName()
+    );
+    if (dryRun) {
+      System.out.println(eventStreamCommand);
+      System.out.println(versionTableCommand);
+    } else {
+      final KsqlRestClient client = KsqlRestClient.create(
+          properties.getKsqlServerUrl(),
+          Collections.EMPTY_MAP,
+          Collections.EMPTY_MAP,
+          Optional.empty()
+      );
+      final RestResponse<KsqlEntityList> streamResponse =
+          client.makeKsqlRequest(eventStreamCommand);
+      final RestResponse<KsqlEntityList> tableResponse =
+          client.makeKsqlRequest(versionTableCommand);
+
+      if (streamResponse.isSuccessful()) {
+        System.out.println("Successfully created migration stream");
+      } else {
+        System.out.println("Failed to create migration stream: "
+            + streamResponse.getErrorMessage());
+      }
+
+      if (tableResponse.isSuccessful()) {
+        System.out.println("Successfully created migration table");
+      } else {
+        System.out.println("Failed to create migration table: "
+            + tableResponse.getErrorMessage());
+      }
+    }
+  }
```

Comment:
I recommend this output for the tool:
```
$ ksql-migrations initialize
Creating stream: "migration_events"
Successfully created migration stream.
Creating table: "schema_version"
Successfully created migration table.

Schema metadata initialized successfully (execution time 00:00.05s).
```
We should add the execution time at the end. This lets users know how much time their migration takes. 
We should print the action prior to executing it so in case the tool is stuck in a request, the user knows where. Otherwise, they will see an empty output and not knowing if the tool is making progress.

If it fails, then we could display:
```
$ ksql-migrations initialize
Creating stream: "migration_events"
Failed to created stream: blablablablabla
```